txnkv/transaction/pipelined_flush.go

```diff
@@ -306,7 +328,12 @@ func (c *twoPhaseCommitter) commitFlushedMutations(bo *retry.Backoffer) error {
 	c.mu.RLock()
 	c.mu.committed = true
 	c.mu.RUnlock()
-	logutil.BgLogger().Info("[pipelined dml] transaction is committed")
+	logutil.BgLogger().Info(
+		"[pipelined dml] transaction is committed",
+		zap.Uint64("startTS", c.startTS),
+		zap.Uint64("commitTS", commitTS),
```

Comment:
Maye not relate to this PR, the `c.commitTS` and  `atomic.LoadUint64(&c.commitTS)` are both used in different places, we need to unify the usages.